calculator/operations.py

```diff
@@ -1,23 +1,28 @@
-
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    total = args[0]
```

Comment:
An alternative way operations could be cleaned up would be by making use of operator library and reduce. Though that's also a style choice if you want to use reduce